src/styles/base.scss

```diff
@@ -0,0 +1,12 @@
+body {
+  margin: 0;
+}
+
+section {
+  margin-top: 1px;
+  margin-left: 1px;
+  width: 198px;
+  height: 406px;
+  border: 1px solid $white_color;
+  border-radius: $border-radius;
```

Comment:
Ensure that the variables `$white_color` and `$border-radius` are defined in a separate SCSS variables file, as per the task requirements. This helps in maintaining consistency and ease of updates across styles.